elasticsearch/install.sh

```diff
@@ -5,11 +5,16 @@ set -ex
 mkdir -p ${HOME}
 ln -s /usr/share/elasticsearch /usr/share/java/elasticsearch
 
-/usr/share/elasticsearch/bin/plugin -i com.floragunn/search-guard/0.5.1 -url https://github.com/lukas-vlcek/origin-aggregated-logging/releases/download/v0.1/search-guard-0.5.1.zip
+/usr/share/elasticsearch/bin/plugin install -b com.floragunn/search-guard-ssl/${SG_SSL_VER}
+#/usr/share/elasticsearch/bin/plugin install -b com.floragunn/search-guard-2/2.3.3.3
 
-/usr/share/elasticsearch/bin/plugin -i io.fabric8.elasticsearch/openshift-elasticsearch-plugin/0.15
-
-/usr/share/elasticsearch/bin/plugin -i io.fabric8/elasticsearch-cloud-kubernetes/1.3.0
+# We are currently carrying a patch for sg-2; cannot use normal install
+/usr/share/elasticsearch/bin/plugin install -b file://${HOME}/search-guard-2-2.3.5.6.zip
```

Comment:
This can be carried forward as tech debt so long as there are no other comments on this PR